src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -11,23 +11,23 @@ redirects:
 </Callout>
 
-To manage their users, New Relic organizations can configure one or more authentication domains, which control how users are added to a New Relic account, how they’re authenticated, and more.
+To manage their users, New Relic organizations can configure one or more authentication domains, which control how users are added to a New Relic account, how they're authenticated, and more.
 
-## What is an authentication domain? [#auth-domain-definition]
+## Authentication domains explained [#auth-domain-definition]
 
-An "authentication domain" is a grouping of New Relic users governed by the same user management settings, like [how they're provisioned](#source-users) (added and updated), [how they're authenticated](#authentication) (logged in), [session settings](#session-mgmt), and [how user upgrades are managed](#session-mgmt).
+An **authentication domain** is a grouping of New Relic users governed by the same user management settings, like [how they're provisioned](#source-users) (added and updated), [how they're authenticated](#authentication) (logged in), [session settings](#session-mgmt), and [how user upgrades are managed](##user-upgrade).
 
-When someone creates a New Relic account, the default authentication settings are:
+When you create a New Relic organization, the default authentication settings are:
 
-* Users are manually added to New Relic
-* Users manually log in using their email and password
+* Source of users: your users are added to New Relic using only our user management tools (not third party tools)
+* Authentication: users log in using their email and password
 
-Those default settings would be under one "authentication domain." Another authentication domain might be set up like this:
+Those default settings would be under one authentication domain. If you added another authentication domain, you might set it up like this:
 
-* Users are added and managed from an identity provider using SCIM provisioning
-* Users are logged in using SAML single sign-on (SSO) from an identity provider
+* Source of users: Users are added and managed from a third-party identity provider via SCIM provisioning
+* Authentication: users are logged in using SAML single sign-on (SSO) from an identity provider
 
-When you add users to New Relic, they’re added to a specific authentication domain. Typically organizations have either one or two authentication domains: one for the manual, default methods and one for the methods associated with an identity provider. Learn more in this short video (4:26 minutes):
+When you add users to New Relic, they're always added to a specific authentication domain. Typically organizations have either one or two authentication domains: one for the manual, default methods and one for the methods associated with an identity provider. Learn more in this short video (4:26 minutes):
```

Comment:
I don't think you need "manual, default methods" here, I think it might read better with only one of them